src/components/query/ListTimeSeriesQueryEditor.tsx

```diff
@@ -0,0 +1,96 @@
+  newFormStylingEnabled?: boolean;
+}
+
+const timeSeriesTypes = [
+  {
+    label: 'ALL',
+    value: 'ALL',
+    description: 'All time series data',
+  },
+  {
+    label: 'ASSOCIATED',
+    value: 'ASSOCIATED',
+    description: "The time series is associated with an asset property.",
+  },
+  { label: 'DISASSOCIATED', value: 'DISASSOCIATED', description: "The time series isn't associated with any asset property." },
+];
+
+export const ListTimeSeriesQueryEditorFunction = (props: Props) => {
+
+  const [lastInput, setLastInput] = useState<string>()
+
+  const onAliasPrefixChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { onChange, query } = props;
+    setLastInput("prefix")
+    onChange({ ...query, aliasPrefix: e.target.value });
+  };
+
+  const onAssetIdChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { onChange, query } = props;
+    setLastInput("id")
+    onChange({ ...query, assetId: e.target.value });
+  };
+
+  const onTimeSeriesTypeChange = (sel: SelectableValue<string>) => {
+    const { onChange, query } = props;
+    if (sel.value === 'ALL' && lastInput === "prefix"){
+      onChange({ ...query, timeSeriesType: sel.value as 'ASSOCIATED' | 'DISASSOCIATED' | 'ALL' , assetId: undefined});
+    }
+    else if (sel.value === 'ALL' && lastInput === "id"){
+      onChange({ ...query, timeSeriesType: sel.value as 'ASSOCIATED' | 'DISASSOCIATED' | 'ALL' , aliasPrefix: undefined});
+    } else {
+      onChange({ ...query, timeSeriesType: sel.value as 'ASSOCIATED' | 'DISASSOCIATED' | 'ALL' });
+    }
+    
+  };
+
+  const { query } = props;
+
+  return (
+      <EditorRow>
+        <EditorFieldGroup>
+          <EditorField label="timeSeriesType" htmlFor="timeSeriesType" width={20}>
+            <Select
+              id="timeSeriesType"
+              aria-label="timeSeriesType"
+              options={timeSeriesTypes}
+              value={timeSeriesTypes.find((v: { value: string; }) => v.value === query.timeSeriesType) || timeSeriesTypes[0]}
+              onChange={onTimeSeriesTypeChange}
+              placeholder="Select a property"
+              menuPlacement="auto"
+            />
+          </EditorField>
+          <EditorField label="aliasPrefix" htmlFor="aliasPrefix" width={30}>
+            <Input
+              id="aliasPrefix"
+              aria-label="Alias Prefix"
+              value={query.aliasPrefix}
+              onChange={onAliasPrefixChange}
+              placeholder="Optional: The alias prefix of the time series."
+            />
+          </EditorField>
+          <EditorField label="assetId" htmlFor="assetId" width={30}>
+            <Input
+              id="assetId"
+              aria-label="Asset Id"
+              value={query.assetId}
+              onChange={onAssetIdChange}
+              placeholder="The ID of the asset in which the asset property was created. This can be either the actual ID in UUID format, or else externalId: followed by the external ID, if it has one"
```

Comment:
I think this placeholder text might be better suited for a tooltip, as it gets cut off. Maybe it would be more appropriate to put in a tooltip. I know the `<EditorField>` component lets you specify a tooltip prop. 

Placeholders are better for examples rather than explanations. So maybe here you could put `exampleAssetId` or `externalId: exampleExternalId` or even just `assetid`